src/main/java/io/github/nucleuspowered/nucleus/modules/ban/commands/TempBanCommand.java

```diff
@@ -34,10 +40,27 @@
 @NoCost
 public class TempBanCommand extends CommandBase<CommandSource> {
 
+    @Inject private BanConfigAdapter bca;
     private final String user = "user";
     private final String reason = "reason";
     private final String duration = "duration";
 
+    private final String bypassMaximumLength = PermissionRegistry.PERMISSIONS_PREFIX + "tempban.bypass";
+
+    @Override
+    public Map<String, PermissionInformation> permissionsToRegister() {
+        Map<String, PermissionInformation> m = new HashMap<>();
+        m.put(bypassMaximumLength, new PermissionInformation(Util.getMessageWithFormat("permission.tempban.bypass"), SuggestedLevel.MOD));
```

Comment:
Just `bypass`, or whatever it becomes. It assumes that it takes whatever the base permission for the command is - `nucleus.tempban` in this case, and adds it on.